src/components/coursCard.tsx

```diff
@@ -0,0 +1,51 @@
+    return <div className="w-[400px] h-[300px] border-black rounded-md border-[3px] flex flex-col">
+        {/* card title */}
+        <div className="bg-[#FFDF36] p-3 flex justify-between flex-row-reverse items-center border-black border-[3px] rounded-[4px] border-t-0 border-r-0 border-l-0 basis-1/6">
+            {data.title}
+            { data.level === "سهل" && <p className="rounded-xl border border-black py-1 px-2 bg-[#A2FF93]">{data.level}</p>}
+            { data.level === "متوسط" && <p className="rounded-xl border border-black py-1 px-2 bg-[#FFEE93]">{data.level}</p>}
+            { data.level === "صعب" && <p className="rounded-xl border border-black py-1 px-2 bg-[#FF9393]">{data.level}</p>}
+        </div>
+        {/* card body */}
+        <div className="text-right flex flex-col justify-between basis-2/3 p-3">
+            <p>{data.description}</p>
+            <div>
+                <p className="font-bold text-[30px]">{data.rating}</p>
+                <div className="flex justify-between items-center flex-row-reverse">
+                    <div className="flex flex-row-reverse items-center gap-2">
+                        <div className="flex flex-row-reverse">
+                            {Array.from(Array(5)).map((_, i)=>{
+                                if(i+1 <= data.rating)
+                                    return <FaStar color="#FFDF36" className="w-[30px] h-[30px]"/>;
+                                else
+                                    return <FaRegStar color="#FFDF36" className="w-[30px] h-[30px]"/>;
+                            })}
+                        </div>
+                        <p>({data.nbrOfRaters} تقيم)</p>
+                    </div>
+                    <Image src={Flag} alt="flag" className="w-[30px] h-[30px]"/>
+                </div>
+            </div>
+            <div className="flex flex-row-reverse">
+                <div className="flex -space-x-1 overflow-hidden">
+                    <img className="inline-block h-6 w-6 rounded-full ring-2 ring-white" src="https://images.unsplash.com/photo-1491528323818-fdd1faba62cc?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80" alt=""/>
+                    <img className="inline-block h-6 w-6 rounded-full ring-2 ring-white" src="https://images.unsplash.com/photo-1550525811-e5869dd03032?ixlib=rb-1.2.1&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80" alt=""/>
+                    <img className="inline-block h-6 w-6 rounded-full ring-2 ring-white" src="https://images.unsplash.com/photo-1500648767791-00dcc994a43e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2.25&w=256&h=256&q=80" alt=""/>
+                    <img className="inline-block h-6 w-6 rounded-full ring-2 ring-white" src="https://images.unsplash.com/photo-1472099645785-5658abf4ff4e?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=facearea&facepad=2&w=256&h=256&q=80" alt=""/>
```

Comment:
when you add Reviews in your course class, just iterate over them and display review.user.image

Reply:
it's only a temporary fix for the display as Haytam said in the last meeting, I will work on the solution for this👍🏻

Reply:
yes sure i understand, i'm just trying to save your time to avoid writing many staff that will be deleted when adding real data.

one last comment, i think we need to add tags/interests for courses, @haythamhakim will also do it for users, this will help us recommend courses for user by looking for courses containing user tags.

Reply:
I agree with your last comment, i think we should discuss it with the UI/UX team first

Reply:
@RaoufGhrissi can you please check the changes?